From: Luotao Fu <l.fu@pengutronix.de>
Subject: [patch] add 'variable' to 100dpi and 75dpi fonts.alias

The "variable" font point to helvectica, which only exists in 75dpi and
100dpi directories. Hence if an application queries for that variable it
may fail.

The font package mechanism is complicated, since directory structure on
the installation target is often not synchronic with the order of the font
packages. Various distributions solve this problem differently. Question
to mainline has yet received no answer.

Signed-off-by: Luotao Fu <l.fu@pengutronix.de>
---
 100dpi/fonts.alias |    1 +
 75dpi/fonts.alias  |    1 +
 misc/fonts.alias   |    1 -
 3 files changed, 2 insertions(+), 1 deletion(-)

Index: font-alias-X11R7.0-1.0.1/100dpi/fonts.alias
===================================================================
--- font-alias-X11R7.0-1.0.1.orig/100dpi/fonts.alias
+++ font-alias-X11R7.0-1.0.1/100dpi/fonts.alias
@@ -1,3 +1,4 @@
+variable     -*-helvetica-bold-r-normal-*-*-120-*-*-*-*-iso8859-1
 lucidasans-bolditalic-8 -b&h-lucida-bold-i-normal-sans-11-80-100-100-p-69-iso8859-1
 lucidasans-bolditalic-10 -b&h-lucida-bold-i-normal-sans-14-100-100-100-p-90-iso8859-1
 lucidasans-bolditalic-12 -b&h-lucida-bold-i-normal-sans-17-120-100-100-p-108-iso8859-1
Index: font-alias-X11R7.0-1.0.1/75dpi/fonts.alias
===================================================================
--- font-alias-X11R7.0-1.0.1.orig/75dpi/fonts.alias
+++ font-alias-X11R7.0-1.0.1/75dpi/fonts.alias
@@ -1,3 +1,4 @@
+variable     -*-helvetica-bold-r-normal-*-*-120-*-*-*-*-iso8859-1
 lucidasans-bolditalic-8 -b&h-lucida-bold-i-normal-sans-8-80-75-75-p-49-iso8859-1
 lucidasans-bolditalic-10 -b&h-lucida-bold-i-normal-sans-10-100-75-75-p-67-iso8859-1
 lucidasans-bolditalic-12 -b&h-lucida-bold-i-normal-sans-12-120-75-75-p-79-iso8859-1
Index: font-alias-X11R7.0-1.0.1/misc/fonts.alias
===================================================================
--- font-alias-X11R7.0-1.0.1.orig/misc/fonts.alias
+++ font-alias-X11R7.0-1.0.1/misc/fonts.alias
@@ -1,6 +1,5 @@
 ! $Xorg: fonts.alias,v 1.3 2000/08/21 16:42:31 coskrey Exp $
 fixed        -misc-fixed-medium-r-semicondensed--13-120-75-75-c-60-iso8859-1
-variable     -*-helvetica-bold-r-normal-*-*-120-*-*-*-*-iso8859-1
 5x7          -misc-fixed-medium-r-normal--7-70-75-75-c-50-iso8859-1
 5x8          -misc-fixed-medium-r-normal--8-80-75-75-c-50-iso8859-1
 6x9          -misc-fixed-medium-r-normal--9-90-75-75-c-60-iso8859-1
